src/strings/index.tsx

```diff
@@ -872,6 +872,7 @@ const strings = new LocalizedStrings({
     NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_DESCRIPTION:
       'There are no batches selected with seedlings that are "ready." Select another withdraw purpose or cancel and select a different batch.',
     OUTPLANTS_REQUIRE_READY_SEEDLINGS: 'Outplants require batches with seedlings that are "ready."',
+    EDIT_QUANTITY_DISABLED: 'Currently, you may only input a quantity when the status is "Drying" or “In Storage”.',
```

Comment:
as per @chudo76 's earlier comments, the "." should be within the quotes.
or "In Storage."

Reply:
😮